dcpy/utils/geospatial.py

```diff
@@ -22,6 +40,72 @@
 }
 
 
+def convert_to_geodata(
+    data: pd.DataFrame,
+    geometry_format: str,
+    geometry_column: str,
+    new_geometry_column: str = "geometry",
+) -> gpd.GeoDataFrame:
+    data = data.copy()
+    match geometry_format:
+        case "WKT":
+            data[geometry_column] = gpd.GeoSeries.from_wkt(data[geometry_column])
+        case "WKB":
+
+            def _try_wkb(wkb):
+                try:
+                    return pd.Series([shapely.from_wkb(wkb), None])
```

Comment:
this shouldn't be a return here, right? but some sort of variable assignment

Reply:
since it's used in a `DataFrame.apply` to create two new columns, I think it has to return rather than assign